pvlib/spectrum/spectrl2.py

```diff
@@ -0,0 +1,384 @@
+r"""
+The ``spectrl2`` module implements the Bird Simple Spectral Model.
+"""
+
+import pvlib
+from pvlib.tools import cosd
+import numpy as np
+import pandas as pd
+
+# SPECTRL2 extraterrestrial spectrum and atmospheric absorption coefficients
+_SPECTRL2_COEFFS = np.zeros(122, dtype=np.dtype([
+    ('wavelength', 'float64'),
+    ('spectral_irradiance_et', 'float64'),
+    ('water_vapor_absorption', 'float64'),
+    ('ozone_absorption', 'float64'),
+    ('mixed_absorption', 'float64'),
+]))
```

Comment:
There seems to be little benefit to putting the constants into a structured array.  How about 5 constant column vectors like this?

```
WAVELENGTH = np.transpose(np.array([[
                   300.0, 305..0 etc.
]]))
```
A side benefit is that one can recognize the use of the constants in the code more easily because they are CAPS.